### PR TITLE
Add bastion ping

### PR DESCRIPTION
**Pull request by jtaleric**
Test connectivity between Prow and our bastion host

### CODE REVIEW

File: ci-operator/step-registry/openshift-qe/installer/bm/ping/openshift-qe-installer-bm-ping-commands.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+bastion=$(cat "/secret/hostname")
+
+ping -c 5 $bastion
```
Comment (vishnuchalla) [inline ci-operator/step-registry/openshift-qe/installer/bm/ping/openshift-qe-installer-bm-ping-commands.sh:9]: Looks like just `hostname` is being used to do a ping. Are we using ssh key as well?

Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/installer/bm/ping/openshift-qe-installer-bm-ping-commands.sh:9]: Why do we need to use the sshkey to ping?

**Review by krishvoor — APPROVED** [review]
/lgtm

**Comment by svetsa-rh** [comment]
@jtaleric can we kick off a test prior to merging?

**Pull request merged**